### ISSUE TITLE
Dataset sectors object not updated after sector deletions

### ISSUE THREAD

**Issue by thomasstjerne** [CatalogueOfLife/backend#771]
https://api.dev.catalogue.life/dataset/2067/tree?catalogueKey=2067&type=CATALOGUE&limit=1000&offset=0
<img width="503" alt="Screenshot at Jun 26 15-07-26" src="https://user-images.githubusercontent.com/1666484/85860387-06445700-b7bf-11ea-96c8-436839baeadc.png">

Gives empty result (no sectors):
https://api.dev.catalogue.life/dataset/2067/sector?datasetKey=2067&limit=100&offset=0

**Comment by mdoering** [comment]
Indeed. This only happens for the new delete which does not remove the taxa, the full delete changes the sectors counts

**Comment by mdoering** [comment]
hm, no. It is changed in the code. Needs more investigations, maybe sth went wrong
